Deque/deque.py

```diff
@@ -49,28 +50,50 @@ def pop_back(self):
         self.tail = ex_tail.prev
         return ex_tail.data
 
-    def deque_to_list(self):
+    def get_front(self):
+        return self.head.data
+
+    def get_back(self):
+        return self.tail.data
+
+    def to_list(self):
         output = list()
         curr = self.head
         while curr is not None:
             output.append(curr.data)
             curr = curr.next
         return output
 
+    def is_empty(self):
+        if self.head is None:
+            print('Deque is Empty')
+        else:
+            print('Deque is not Empty')
+        return
```

Comment:
"is_empty" sounds like a function that returns a boolean value. 
So it should return True if deque is empty, and False if not.

Reply:
fixed